### ISSUE TITLE
Move CI from Travis to Jenkins

### ISSUE THREAD

**Issue by zenorocha** [wedeploy/wedeploy-sdk-swift#26]

**Comment by victorg1991** [comment]
Do we have machines running macOS in the CI?

**Comment by mdelapenya** [comment]
No, we haven't, and this is a serious concern that we have to address before starting with this move.

Maybe we can keep OS-based projects on Travis

**Comment by tmoreira2020** [comment]
Let's leave it as the last project to move, until there we research how to run OSx on Jenkins

**Comment by mdelapenya** [comment]
The only problem is getting the Mac machines. We could invest in a Mac mini for that 🤔 

https://blog.samsao.co/how-to-setup-a-jenkins-slave-running-mac-os-x-for-ios-projects-part-1-2937502ce90b

**Comment by victorg1991** [comment]
If this causes a lot of troubles, I think it should be enough using the OSS free version of Travis
